src/content-handlers/iiif/modules/uv-shared-module/LeftPanel.ts

```diff
@@ -32,6 +32,9 @@ export class LeftPanel<
       this.options.panelOpen
     );
 
+    const viewportWidth: number = document.documentElement.clientWidth;
+    if(viewportWidth <= 767) return;
```

Comment:
I like the functionality that this provides, but I wonder if we can implement it in a more flexible way that avoids hard-coded magic numbers. Might it make sense to replace the panelOpen boolean configuration setting with a panelOpenMinWidth integer setting? Then it could be 0 to always open, some incredibly high number to always close, or a configurable breakpoint the rest of the time. What do you think? @edsilv?

Reply:
@demiankatz One possibility that avoids any config stuff would be to assume that this behaviour is only needed when it's a portrait-oriented, touch-enabled device, which I think in 99% of cases would be a phone/tablet?

Reply:
That's true, but I think @edsilv's point was that displaying the sidebar on mobile isn't the ideal solution to the problem, regardless of what approach we take for deciding to do it. Though if it's an urgent local need, I think the approach here might be able to be locally adopted as a workaround until the more complete solution is ready.